src/material-examples/popover-edit-mat-table/popover-edit-mat-table-example.ts

```diff
@@ -50,10 +50,17 @@ export class PopoverEditMatTableExample {
   readonly preservedNameValues = new WeakMap<PeriodicElement, any>();
   readonly preservedWeightValues = new WeakMap<PeriodicElement, any>();
 
-  constructor(iconRegistry: MatIconRegistry, sanitizer: DomSanitizer) {
-    iconRegistry.addSvgIcon(
-        'edit',
-        sanitizer.bypassSecurityTrustResourceUrl('assets/img/examples/edit-icon.svg'));
+  constructor(
+      private readonly _snackBar: MatSnackBar,
+      iconRegistry: MatIconRegistry,
+      sanitizer: DomSanitizer) {
+    iconRegistry
+        .addSvgIcon(
+            'edit',
+            sanitizer.bypassSecurityTrustResourceUrl('/popover-edit/assets/edit-icon.svg'))
+        .addSvgIcon(
+            'popover-edit-thumb-up',
+            sanitizer.bypassSecurityTrustResourceUrl('/popover-edit/assets/thumbup-icon.svg'));
```

Comment:
Are these icons not in the icon font?

Reply:
Just using mat-icon the "normal" way was not working for me, but maybe I was doing something wrong?

Reply:
When I put `<mat-icon>thumb_up</mat-icon>` into `src/material-examples/popover-edit-mat-table/popover-edit-mat-table-example.html` locally it seems to work

Reply:
Yup - I was probably doing thumb-up.

Svg stuff removed